backend/utils/authutils/path.go

```diff
@@ -0,0 +1,72 @@
+package authutils
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+)
+
+var getOS = func() string {
+	return runtime.GOOS
+}
+
+var getUserHomeDir = os.UserHomeDir
```

Comment:
instead of using the os.UserHomeDir, i think it's preferable to use the [XDG standard](https://en.wikipedia.org/wiki/Freedesktop.org#Base_Directory_Specification) so files reside in standardized places for linux + OSX. 

in golang i've used this module for getting xdg paths across OS's (including windows): https://github.com/adrg/xdg. there's brief usage instructions at https://pkg.go.dev/github.com/adrg/xdg#readme-usage

**data example**

store the data using the base path returned by `xdg.XDG_DATA_HOME` or, for example 
```
xdg.DataFile(filepath.Join("Tella", ".tvault"))
``` 
**config example**

store any tella desktop configuration using the base path returned by `xdg.XDG_CONFIG_HOME` or 

```
xdg.ConfigFile(filepath.Join("Tella", ".tella-config"))
```

Reply:
ohh didn't know about this. will look into it